docs_nnx/guides/bridge_guide.md

```diff
@@ -494,48 +604,37 @@ class NNXDotWithParititioning(nnx.Module):
     return x @ self.w
 
 x = jax.random.normal(jax.random.key(42), (4, 32))
-
-@jax.jit
-def create_sharded_variables(key, x):
-  model = bridge.to_linen(NNXDotWithParititioning, 32, 64)
-  variables = model.init(key, x)
-  # A `NNXMeta` wrapper of the underlying `nnx.Param`
-  assert type(variables['params']['w']) == bridge.NNXMeta
-  # The annotation coming from the `nnx.Param` => (in, out)
-  assert variables['params']['w'].metadata['sharding'] == ('in', 'out')
-
-  unboxed_variables = nn.unbox(variables)
-  variable_pspecs = nn.get_partition_spec(variables)
-  assert isinstance(unboxed_variables['params']['w'], jax.Array)
-  assert variable_pspecs['params']['w'] == jax.sharding.PartitionSpec('in', 'out')
-
-  sharded_vars = jax.tree.map(jax.lax.with_sharding_constraint,
-                              nn.unbox(variables),
-                              nn.get_partition_spec(variables))
-  return sharded_vars
+model = bridge.to_linen(NNXDotWithParititioning, 32, 64)
```

Comment:
Code changed significantly in the last doc revision, will revert back to the new code `@jit def create_sharded_variables(...)...` @cgarciae